components/supervisor-api/terminal.proto

```diff
@@ -69,6 +71,8 @@ message ListTerminalsResponse {
         string alias = 1;
         repeated string command = 2;
         string title = 3;
+        int64 pid = 4;
```

Comment:
Do you think it would be possible as well to provide `cwd`?  It is needed for `Attach Session` command to display in detail description of the quick pick. There are also 2 API requests from Code: one to get current cwd and another initial.

Reply:
We could certainly add cwd. How important is the initial cwd?

Reply:
I think I can store initial cwd in attributes as well, so providing just current cwd should be fine. Initial cwd is only  used to resolve links in terminal till effective cwd is available.